packages/wrangler/src/dev.ts

```diff
@@ -366,6 +338,12 @@ type DevArguments = StrictYargsOptionsToInterface<typeof devOptions>;
 export async function devHandler(args: DevArguments) {
 	await printWranglerBanner();
 
+	if (args.experimentalDevEnv) {
+		logger.warn(
+			"--x-dev-env is no longer required and will be removed in a future version.\n`wrangler dev` now uses the unified dev server by default. 🎉"
```

Comment:
will users know what "unified dev server" means, or is it even relevant for them? Maybe specifying that `--x-dev-env` is now on by default is enough?

Reply:
That's a fair point! I'll change that 👍

Reply:
https://github.com/cloudflare/workers-sdk/pull/7069/commits/6ffd3efafa05aee5303e4b5f5c5f3833aed290f4